### PR TITLE
macho: Enable debugging stripped binaries

### PR DESCRIPTION
**Pull request by pgavlin**
Load a symbol table from pclntab as when debugging stripped ELF binaries.

### CODE REVIEW

File: pkg/proc/proc_test.go [hunk 1]
```diff
@@ -3179,8 +3179,8 @@ func TestShadowedFlag(t *testing.T) {
 
 func TestDebugStripped(t *testing.T) {
 	// Currently only implemented for Linux ELF executables.
-	// TODO(derekparker): Add support for Mach-O and PE.
-	skipUnlessOn(t, "linux only", "linux")
+	// TODO(derekparker): Add support for PE.
+	skipOn(t, "not working on windows", "windows")
```
Comment (aarzilli) [inline pkg/proc/proc_test.go:3183]: You also need to skip it on freebsd.

**Review by aarzilli — COMMENTED** [review]
The backend test health file needs to be regenerated: `go run _scripts/gen-backend_test_health.go`.

**Comment by pgavlin** [comment]
All feedback addressed; PTAL

**Review by aarzilli — APPROVED** [review]
LGTM

**Review by derekparker — APPROVED** [review]
lgtm

**Pull request merged**